core/txpool/legacypool/list.go

```diff
@@ -456,7 +458,7 @@ func (l *list) LastElement() *types.Transaction {
 // total cost of all transactions.
 func (l *list) subTotalCost(txs []*types.Transaction) {
 	for _, tx := range txs {
-		l.totalcost.Sub(l.totalcost, tx.Cost())
+		l.totalcost.Sub(l.totalcost, uint256.MustFromBig(tx.Cost()))
```

Comment:
This call to `Sub` can now overflow, which it could not previously. This can be handled in a couple of ways: 

- Ignore it (what this PR does)
- Set to zero on overflow, instead of near infinite
- `panic` on overflow

Not sure what approach is the best, but it's definitely a point to consider

Reply:
I think going with `panic` on overflow is the best approach here. Considering the current uses of `subTotalCost`, it subtracts the total cost of previously added txs, so it should not be possible to underflow.